metabrainz/templates/users/tier.html

```diff
@@ -1,9 +1,9 @@
 {% extends 'base.html' %}
 
-{% block title %}{{ tier.name + _('tier') }} - MetaBrainz Foundation{% endblock %}
+{% block title %}{{ _(tier.name + 'tier') }} - MetaBrainz Foundation{% endblock %}
```

Comment:
you're right. I tried generating the po files with this `{{ _(tier.name + 'tier') }}` and this string was not there.

on trying with `$(name)s tier` , it works. I'll submit the fix